### ISSUE TITLE
fix sorting by clicking on column 

### ISSUE THREAD

**Issue by AgnitumuS** [vitmalina/w2ui#847]
### On 1.5 Exists one bug:

if set 
" selectType : 'cell' "

We lost sorting functionality.

to fix i found one solution: 
in file w2ui.js
on function columnClick add before (line 4511)

``` javascript
} else {
                    if (!event.shiftKey && !event.metaKey && !event.ctrlKey) {
                        this.selectNone();
                    }
                    var tmp     = this.getSelection();
                    var column  = this.getColumn(eventData.field, true);
```

add

``` javascript
if (event.altKey){
                        var column = this.getColumn(field);
                        if (column && column.sortable) this.sort(field, null, (event && (event.ctrlKey || event.metaKey) ? true : false) );
                        return;
                    }
```

After this we have able Using sorting on click + Alt

**Comment by mpf82** [comment]
Fixed in PR #1705

@mrg2001 / @vitmalina  This issue can probably be closed.

**Comment by mpf82** [comment]
Closing issue: fixed
